apps/performance-tests/src/util/BlockHandler.ts

```diff
@@ -21,7 +33,8 @@ export interface Summary {
  */
 export class BlockHandler {
   private readonly _samples = new SortedArray<number>((a, b) => a - b);
-  private _timer?: NodeJS.Timeout;
+  private _running = false;
```

Comment:
could we use `_promise` as the indication for a running timer?

Reply:
That's right. Made the suggested change